test/utils.jl

```diff
@@ -73,16 +73,16 @@ end
     @test_nowarn similar(ps_c, 10)
     @test_nowarn similar(ps_c)
 
-    ps_c_f, ps_c_re = Functors.functor(ps_c)
-    @test ps_c_f == ps
-    @test ps_c_re(ps_c_f) == ps_c
+    #ps_c_f, ps_c_re = Functors.functor(ps_c) # Comment out since we probably don't need this
```

Comment:
We should still be testing this I think

Reply:
I have deleted the overload of `Functors.functor`, I feel like it's not necessary anymore, so the testing here would fail.

Reply:
I can be incredibly wrong tho